scripts/sync-malicious-packages/sync-malicious-packages.py

```diff
@@ -16,6 +19,28 @@ def parse_arguments():
     parser.add_argument('--dynamodb-table', help='DynamoDB table containing the scan results')
     args = parser.parse_args()
     return args
+
+
+def update_manifest(destination, samples):
+  manifest_path = os.path.join(destination, "manifest.json")
+
+  with open(manifest_path, 'r') as f:
+    manifest = json.loads(f.read())
+
+  for sample in samples:
+    package, version = sample["package_name"], sample["package_version"]
+    if package.startswith("npm|"):
```

Comment:
do we need to handle Go packages similarily?

Reply:
We aren't currently monitoring the Golang ecosystem, but if that ever happens, we would need to update this line.

Reply:
indeed